include/csoundCore.h

```diff
@@ -317,12 +317,11 @@ typedef struct CORFIL {
         char    *opname;
         uint16  dsblksiz;
         uint16  flags;
-        uint8_t thread;
         char    *outypes;
         char    *intypes;
-        int32_t     (*iopadr)(CSOUND *, void *p);
-        int32_t     (*kopadr)(CSOUND *, void *p);
-        int32_t     (*aopadr)(CSOUND *, void *p);
+        int     (*init)(CSOUND *, void *p);
```

Comment:
Any reason for the move from int32_t to int?

Reply:
no. fixed